src/channel.js

```diff
@@ -158,6 +158,38 @@ export class Channel {
 		});
 	}
 
+	/**
+	 * search - Query Members
+	 *
+	 * @param {object|string}  filterConditions object MongoDB style filters
+	 * @param {object} options        Option object, {limit: 10, offset:10}
+	 *
+	 * @return {object} search members response
+	 */
+	async queryMembers(filterConditions, options = {}) {
+		if (!options) {
+			options = {};
+		}
+		let id;
+		const type = this.type;
+		let members;
+		if (this.id) {
+			id = this.id;
+		} else if (this.data && Array.isArray(this.data.members)) {
+			members = this.data.members;
```

Comment:
Why do we need this? (just curious) @thesyncim

Reply:
this is needed for member-based channels (channels without ID)

Reply:
are you referring to one-on-one conversations? Because they also have some id - `"!members-TAMXTZLQ5V6tHNhmyYupyOwNRuWyxShsqdLhxgk7jLc"` Can't we use that?

Reply:
you could also use only the ID, but that would not work for calls like this one.
`channel('messaging',{members:[a,b,c]}).queryMembers({})` (id is undefined at this point)